server/api/graphql/graphqlSchema.js

```diff
@@ -0,0 +1,226 @@
+// Import type helpers from grapghql-js
+import { GraphQLSchema, GraphQLObjectType, GraphQLInt, GraphQLString,
+  GraphQLList, GraphQLID } from "graphql";
+
+// Import some project data collections
+import {Accounts, Orders, Products, Shops} from "/lib/collections";
+
+const OrderItems = new GraphQLObjectType({
+  name: "Ordered",
+  description: "Details of Order Made",
+  fields: () => ({
+    title: { type: GraphQLString },
+    quantity: { type: GraphQLString },
+    price: {
+      type: GraphQLString,
+      resolve: (obj) => {
+        return obj.variants.price;
+      }
+    }
+  })
+});
+
+const OrdersType = new GraphQLObjectType({
+  name: "Orders",
+  description: "Order Status Details",
+  fields: () => ({
+    deliveryAddress: {
+      type: ShippingAddress,
+      resolve: (obj) => {
+        return obj.shipping[0].address;
+      }
+    },
+    email: { type: GraphQLString },
+    _id: { type: GraphQLID },
+    items: { type: new GraphQLList(OrderItems) },
+    userId: { type: GraphQLString },
+    workflowStatus: {
+      type: GraphQLString,
+      resolve: (obj) => {
+        return obj.workflow.status;
+      }
+    },
+    sessionId: { type: GraphQLString },
+    shopId: { type: GraphQLString },
+    orderDate: {
+      type: GraphQLString,
+      resolve: (obj) => {
+        return obj.createdAt;
+      }
+    },
+    shipped: {
+      type: GraphQLString,
+      resolve: (obj) => {
+        return obj.shipping[0].shipped;
+      }
+    },
+    tracking: {
+      type: GraphQLString,
+      resolve: (obj) => {
+        return obj.shipping[0].tracking;
+      }
+    }
+  })
+});
+
+const ProductsType = new GraphQLObjectType({
+  name: "Products",
+  description: "Lists select Product fields",
+  fields: () => ({
+    title: { type: GraphQLString },
+    _id: { type: GraphQLString },
+    vendor: { type: GraphQLString },
+    createdAt: {
+      type: GraphQLString,
+      resolve: (obj) => {
+        return obj.createdAt;
```

Comment:
@andela-wmaina @resh you are right, using the resolve is redundancy in cases where the field name is `createdAt`. Rectified and changes pushed.